src/lib/operators/abstract_read_write_operator.hpp

```diff
@@ -32,7 +32,7 @@ class AbstractReadWriteOperator : public AbstractOperator,
   explicit AbstractReadWriteOperator(const std::shared_ptr<const AbstractOperator> left = nullptr,
                                      const std::shared_ptr<const AbstractOperator> right = nullptr);
 
-  std::shared_ptr<AbstractOperator> recreate(const std::vector<AllParameterVariant>& args) const final;
+  std::shared_ptr<AbstractOperator> recreate(const std::vector<AllParameterVariant>& args) const override;
```

Comment:
Can be removed, I think? Or are there any ReadWriteOperators without recreate left?